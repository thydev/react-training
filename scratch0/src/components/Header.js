import React, { Component } from 'react';

const Header = () => (
  <header className="px4">
    <h1 className="h1">&#x1F372; My Recipes</h1>
  </header>
);

export default Header;
