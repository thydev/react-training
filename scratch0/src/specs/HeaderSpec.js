import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Header from '../components/Header';

describe('<Header />', () => {
  test('Should render correctly', () => {
    const component = renderer.create(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    );
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
