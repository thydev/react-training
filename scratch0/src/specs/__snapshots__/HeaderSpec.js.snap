// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Should render correctly 1`] = `
<header
  className="flex items-center justify-between px4"
>
  <h1
    className="h1"
  >
    <span
      role="img"
    >
      üç≤
    </span>
     My Recipes
  </h1>
  <nav>
    <a
      aria-current="page"
      className="p1 mx2 text-decoration-none rounded bg-white"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      className="p1 mx2 text-decoration-none rounded"
      href="/favorites"
      onClick={[Function]}
    >
      Favorites
    </a>
  </nav>
</header>
`;
